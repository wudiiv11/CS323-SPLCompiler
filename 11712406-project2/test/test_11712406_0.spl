int foo()
{
  int a = 10;
  return a;
}

int test_2_o01(int a, int b)
{
  int c;
  c = foo();
  if (b > 0)
  {
    int a = c + 3;
    3 = b - a;
  }
  a = a[5] + b;
  return a;
}