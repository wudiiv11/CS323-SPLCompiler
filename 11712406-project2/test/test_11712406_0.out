Error type 6 at Line 14: left side in assignment is rvalue
Error type 10 at Line 16: indexing on non-array variable
Error type 5 at Line 16: unmatching type on both sides of assignment